tests/Aspire.Templates.Tests/NewUpAndBuildStandaloneTemplateTests.cs

```diff
@@ -23,24 +23,14 @@ public async Task CanNewAndBuild(string templateName, TestSdk sdk, TestTargetFra
             _ => throw new ArgumentOutOfRangeException(nameof(sdk))
         };
 
-        var templateHive = templates switch
```

Comment:
Can you help me understand this change? Is it that we are now doing this work outside of the test execution so no longer to setup the custom hive here?

Reply:
We have one custom hive now, instead of multiple where we installed 8.0, 9.0, and 8.0+9.0 templates.